utonics/labproject/main.go

```diff
@@ -265,16 +288,18 @@ func newProject(values map[string][]string, botClient, userClient *worker.Client
 		msgs = append(msgs, fmt.Sprintf("Upload failed: %s", err.Error()))
 		return msgs, err
 	}
+	
+	  msgs = append(msgs, "submodule content cannot be initialised and therefore pushed, yet. please initialise with synchronisation script.")
 
-	for _, submodule := range submodules {
-		os.Chdir(submodule.path)
-		msgs = append(msgs, "Uploading submodule to new project repository")
-		if err := uploadProjectRepository(botClient, remoteName); err != nil {
-			msgs = append(msgs, fmt.Sprintf("Upload failed: %s", err.Error()))
-			return msgs, err
-		}
-		os.Chdir("..")
-	}
+//for _, submodule := range submodules {
+//		os.Chdir(submodule.path)
+//		msgs = append(msgs, "Uploading submodule to new project repository")
+//		if err := uploadProjectRepository(botClient, remoteName); err != nil {
+//			msgs = append(msgs, fmt.Sprintf("Upload failed: %s", err.Error()))
+//			return msgs, err
+//		}
+//		os.Chdir(localRepoPath)
+//	}
```

Comment:
If the code isn't needed, just delete it.  It'll still be available in the revision history if it needs to be brought back.

Reply:
Linked to l 247 problem, if the gin init would be implemented, this would come back.

it is commented out because it fails wihout initiation, but it is still something we will want back at some point.